packages/material-ui/src/RangeWidget/RangeWidget.tsx

```diff
@@ -36,10 +36,10 @@ const RangeWidget = ({
       </FormLabel>
       <Slider
         disabled={disabled || readonly}
-        onChange={_onChange}
+        onChange={(_onChange as unknown) as OnSliderChangeEvtHdlr}
```

Comment:
This can be reverted, no?

Reply:
Same as above. TS typing error. If you're not seeing these, then there must be something different between our dev environments. I'm running Typescript v4.2.3. Maybe that's the difference?

Reply:
Ah... `packages/material-ui` is using `3.5.3`

Reply:
@rollercodester Let's revert this change, but reference it a new issue about upgrading typescript 3 to 4.

The reason why is because I'm not sure defining `OnSliderChangeEvtHdlr` (and casting _onChange as unknown before casting it back) is the cleanest way to do it -- we might be able to change the signature / type of `_onChange` directly, for example, so it can accept the additional parameters of `OnSliderChangeEvtHdlr`. If you can find a way to do that, then I'd also be fine if you included that change in this PR.